src/Composer/Installers/EzPlatformInstaller.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Composer\Installers;
+
+class EzPlatformInstaller extends BaseInstaller
+{
+    protected $locations = array(
+        'meta-assets' => 'web/assets/ezplatform/',
```

Comment:
well no, it's on purpose for this type of package. `meta-assets` package are supposed to contain (bower) packages and installed directly there so that from `assets` packages, it's like if everything was installed with bower.